<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry"><rect><x>0</x><y>0</y><width>800</width><height>500</height></rect></property>
  <property name="windowTitle"><string>Информация о кофе</string></property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton">
    <property name="geometry"><rect><x>200</x><y>10</y><width>120</width><height>30</height></rect></property>
    <property name="text"><string>Добавить запись</string></property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry"><rect><x>400</x><y>10</y><width>120</width><height>30</height></rect></property>
    <property name="text"><string>Редактировать запись</string></property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry"><rect><x>20</x><y>50</y><width>760</width><height>420</height></rect></property>
   </widget>
  </widget>
 </widget>
</ui>
